modules/services/swayidle.nix

```diff
@@ -108,6 +108,8 @@ in {
 
       Service = {
         Type = "simple";
+        # swayidle executes commands using "sh -c", so the PATH needs to contain a shell.
+        Environment = [ "PATH=${dirOf pkgs.runtimeShell}" ];
```

Comment:
Thinking back at my solution, it might be better to simply use `lib.makeBinPath [ pkgs.bash ]` instead of the `dirOf` "hack".
What do you think @jneem @sumnerevans?

Reply:
I guess it only needs to have access to to the `sh` command, eh? I think just putting `bash` into the `PATH` would be sufficient then.

Reply:
Sure, that seems to work. I've also rebased on latest master.